frontend-new/pages/main/components/LinkActions.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState, MouseEvent, FC, useCallback } from 'react'
```

Comment:
Let's keep all components flat under the `components` folder it will make easier to find them. 
We should strive for reusability.

Reply:
Agree, also thought about that